padle/padle.py

```diff
@@ -0,0 +1,485 @@
+                    await ctx.send(msg)
+                    await ctx.send("*Hint: You can use the command `^padle score` anywhere "
+                                   "and have Tsubaki automatically share your score for you!*")
+                else:
+                    await ctx.message.delete()
+                    msg = ctx.author.mention + "'s PADle #{}: {}".format(
+                        await self.config.numDays(), len(await self.config.user(ctx.author).guesses())) \
+                        + "\n" + "".join(score)
+                    await ctx.send(msg)
+        else:
+            await ctx.send("You have not done today's PADle yet!")
+
+    async def generatePadle(self):
+        async def is_day_change():
+            curDay = datetime.datetime.now().day
+            oldDay = await self.config.storedDay()
+            if curDay != oldDay:
+                await self.config.storedDay.set(curDay)
+                return True
+
+        await self.bot.wait_until_ready()
+        # idk what x is but it wants it to be there
+        async for x in conditional_iterator(is_day_change, poll_interval=10):
+            try:
+                # to be used at a later date...
+                async with self.config.saveDailyScores() as saveDaily:
+                    saveDaily.append([await self.config.padleToday(), await self.config.allScores()])
+                with open("./pad-cogs/padle/monsters.txt", "r") as f:
+                    monsters = f.readline().split(",")
+                    await self.config.padleToday.set(random.choice(monsters))
+                num = await self.config.numDays()
+                await self.config.numDays.set(num + 1)
+                allUsers = await self.config.all_users()
+                await self.config.allScores.set([])
+                for userid in allUsers:
+                    user = await self.bot.fetch_user(userid)
+                    await self.config.user(user).guesses.set([])
+                    # need to send message if a user is mid-game
+                    if(await self.config.user(user).start() and not await self.config.user(user).done()):
+                        await user.send("The PADle expired; a new one is available.")
+                    await self.config.user(user).start.set(False)
+                    await self.config.user(user).done.set(False)
+                    await self.config.user(user).score.set([])
+                    await self.config.user(user).editID.set("")
+                    await self.config.user(user).channelID.set("")
+                    await self.config.user(user).embedText.set([])
+                subbedUsers = await self.config.subs()
+                for userid in subbedUsers:
+                    user = await self.bot.fetch_user(userid)
+                    await user.send("PADle #{} is now available!".format(await self.config.numDays()))
+            except asyncio.CancelledError:
+                pass
```

Comment:
you probably want to log this? (ask aradia if you need help)

Reply:
ok aradia says that we don't need to log this, so just add a comment saying like

```
# this just means the cog was unloaded or we turned off the bot
pass
```